client/src/components/TaskFilterSelector/TaskFilterSelector.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styles from "./TaskFilterSelector.module.css";
+import { PropTypes } from "prop-types";
+
+const FilterButton = ({ filter, filterSelected, onFilterChanged }) => {
+  return (
+    <button
+      className={
+        filter === filterSelected ? styles.selectedFilter : styles.filterButton
+      }
+      onClick={() => onFilterChanged(filter)}
+    >
+      {filter}
+    </button>
+  );
+};
+
+FilterButton.propTypes = {
+  filter: PropTypes.string.isRequired,
+  filterSelected: PropTypes.string.isRequired,
+  onFilterChanged: PropTypes.func.isRequired,
+};
+
+export const TaskFilterSelector = ({ filterSelected, onFilterChanged }) => {
```

Comment:
I would change the `filterSelected` propName to be `currentFilter` (Just so it's more easily differentiated from just the regular `filter`.)

Once changed, filterSelected should be updated accordingly (to whichever other files use this prop)

Reply:
Renamed.